app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,20 @@
+class CommentsController < ApplicationController
```

Comment:
- [ Required ] According to the project requirements a new comment should be created from a form on this route `/posts/:post_id/comments/new ` as much as your implementation works it would be better if you move the add comment form to this URL as stated in project requirements. 

This means you'll have to create a view file in `views/comments/new.html.erb` which accepts input for creating comments and create a method called `new` in `comments_controller.rb` that renders the form the same way you did for `posts`

## Example code 
```ruby
  def new
    @comment = Comment.new
  end
```